src/test/java/uk/gov/pay/api/service/GetPaymentEventsServiceTest.java

```diff
@@ -28,30 +29,36 @@
     @Rule
     public PactProviderRule connectorRule = new PactProviderRule("connector", this);
 
+    @Rule
+    public PactProviderRule ledgerRule = new PactProviderRule("ledger", this);
```

Comment:
I am guessing there will be pact test that checks ledger events

Reply:
precisely (if you look at the first commit only you can even see it - I'll reintroduce it in the next PR).